theories/dune

```diff
@@ -0,0 +1,57 @@
+(copy_files %{project_root}/lib/*.elpi)
+(copy_files %{project_root}/etc/*.elpi)
+(copy_files %{project_root}/derive/*.elpi)
+(copy_files %{project_root}/engine/*.elpi)
+(copy_files %{project_root}/ltac/*.elpi)
+
+(install
+ (section lib)
+ (package coq-elpi)
+ (files
+   coq-lib.elpi
+   coq-lib-extra.elpi
+   coq-HOAS.elpi
+   coq-elpi-checker.elpi
+   elpi-command.elpi
+   elpi-tactic.elpi
+
+   (bcongr.elpi as derive/bcongr.elpi)
```

Comment:
Am I correct that copy_files puts everything at the root and here we put it back in place?
This seems quite weird.

Reply:
`copy_files` puts stuff in the current dir, then I choose to install in a sub-folder as originally; indeed this could use some tweaking.